src/ape/api/providers.py

```diff
@@ -316,12 +316,18 @@ def send_call(self, txn: TransactionAPI, **kwargs) -> bytes:  # Return value of
         """
 
     @abstractmethod
-    def get_receipt(self, txn_hash: str) -> ReceiptAPI:
+    def get_receipt(
+        self, txn_hash: str, required_confirmations: int = 0, timeout: Optional[int] = None
+    ) -> ReceiptAPI:
         """
         Get the information about a transaction from a transaction hash.
 
         Args:
             txn_hash (str): The hash of the transaction to retrieve.
+            required_confirmations (int): The amount of block confirmations
+              to wait before returning the receipt. Defaults to ``0``.
+            timeout (Optional[int]): The amount of time to wait for a receipt
```

Comment:
We should what it means to be `None`.
I think `None` means use the settings block time out thing? Or does it mean to never timeout?

Reply:
Is there a default timeout?"

Reply:
When `None` the default value returns from the `transaction_acceptance_timeout` setting.